### DISCUSSION TITLE
使用5.x二维码的时候报错

### DISCUSSION THREAD

**Discussion by sunshineXin111** [ant-design/ant-design#47951] [Q&A] [answered]
报错信息 
Module not found: Error: Can't resolve 'antd/es/q-r-code/style' in 'xxxxxxxxx'

**Comment by zombieJ** [comment, marked as answer]
不要用 `babel-plugin-import` 插件

**Reply by sunshineXin111** [reply]
这个插件需要删除么

**Reply by yoyo837** [reply]
看一下文档，搜一下issue, 一堆。
